README.md

```diff
@@ -26,28 +27,59 @@ Anvil is a small Java library for creating reactive user interfaces. Originally
 * Fast (uses no reflection¹)
 * Efficient (views are updated lazily, if the data change didn't affect the view - it remains untouched)
 * Easy to read declarative syntax
-* Java 8 and Kotlin-friendly, but supports Java 6 as well
+* Powerfull typesafe dsl
 * XML layouts are supported, too
 
 ¹Reflection is still used to inflate views once (standard XML inflater does the
 same thing, so no performance loss here).
 
 ## Installation
 
+With Kotlin Gradle DSL
 ```kotlin
-// build.gradle.kts
 repositories {
     maven(url = "https://dl.bintray.com/inkremental/maven")
 }
 dependencies {
-    implementation("dev.inkremental:anvil-sdk17:0.8.0")
+    //For Lollipop as min SDK target 
+    implementation("dev.inkremental:anvil-sdk:21-$latestReleaseVersion")
+    //For Kitkat as min SDK target
+    //implementation("dev.inkremental:anvil-sdk:19-$latestReleaseVersion") 
+    //For ICS as min SDK target
+    //implementation("dev.inkremental:anvil-sdk:17-$latestReleaseVersion") 
+    implementation("dev.inkremental:anvil-support-v4:1.1.0-$latestReleaseVersion") 
+    implementation("dev.inkremental:anvil-appcompat-v7:1.1.0-$latestReleaseVersion")
+    implementation("dev.inkremental:anvil-recyclerview-v7:1.1.0-$latestReleaseVersion")
+    implementation("dev.inkremental:anvil-design:1.1.0-$latestReleaseVersion")
+    implementation("dev.inkremental:anvil-cardview-v7:1.0.0-$latestReleaseVersion")
+    implementation("dev.inkremental:anvil-constraintlayout:1.1.3-$latestReleaseVersion")
```

Comment:
Oh, an additional thing: as we have our dependencies broken, we have to specify both androidx and our artifacts at the same time right now. Let's show that in README and remove a bit later once I fix the fricking publishing.